src/test/java/org/example/C01_01_PrintNameDateOfBirtMobileNbTest.java

```diff
@@ -0,0 +1,36 @@
+package org.example;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class C01_01_PrintNameDateOfBirtMobileNbTest {
```

Comment:
This tests are not ok.
The correct test is:
```
    @Test
    public void personDetails() {

        String name = "Diana-Lavinia";
        String dateOfBirth = "16.09.1989";
        String number = "0730......";
        String person = C01_01_PrintNameDateOfBirtMobileNb.personDetails(name, dateOfBirth, number);
        assertEquals(""""
                Name: Dragoi Diana
                Date of birth: 16.09.1989
                Phone Number: 0730......
        """, person);
    }
```

Reply:
I had some problems with assertEquals under the model above so I've made another variable "expected" where i put all the expected answers and that worked